### DISCUSSION TITLE
ocalhost无法订阅？

### DISCUSSION THREAD

**Discussion by Freddy23317** [DIYgod/RSSHub#16584] [Q&A]
我按照https://docs.rsshub.app/deploy/#docker-compose-deployment-recommended里的提示完成了docker compose的部署，在浏览器输入http://localhost:1200/后显示的结果显示部署没问题
![屏幕截图 2024-08-29 194003](https://github.com/user-attachments/assets/379271f6-40b4-461a-bc0a-e79ae76e35fd)
但但我把域名从.rsshub.app换成http://localhost:1200后，我试着订阅新的rss源，我的rss订阅app提示让我订阅正确的url?这是怎么回事？
![Uploading 屏幕截图 2024-08-29 194544.png…]()

**Comment by flyingicedragon** [comment]
有的RSS订阅软件可能对RSS有限制。例如tt-rss不允许带端口号。看看是不是这个问题。
